src/app.tsx

```diff
@@ -25,10 +24,10 @@ const render = (Component: () => JSX.Element) => {
 render(Application);
 
 // Hot Module Replacement API
-if (typeof module.hot !== "undefined") {
-    module.hot.accept("./components/application", () => {
-        import("./components/application").then((app) => {
-            render(app.default);
-        });
-    });
-}
+// if (typeof module.hot !== "undefined") {
```

Comment:
Either resolve the error caused by this code or discuss if this can be removed with @roelvdboom. There shouldn't be any commented out code.

Reply:
Made a comment on how to get this working in dev. Please uncomment this code.